tools/about_paint.py

```diff
@@ -92,17 +96,37 @@ def _about_paint(paint):
         transform = (tr.xx, tr.yx, tr.xy, tr.yy, tr.dx, tr.dy)
         # just scale and/or translate?
         if tr.xy == 0 and tr.yx == 0:
-            if f2dot14_safe(tr.xx, tr.yy):
-                if tr.dx == 0 and tr.dy == 0:
-                    count_by_type[paint.getFormatName() + "::scale_origin"] += 1
+            # Relationship between scale and translate leads to div 0?
+            if ((tr.xx == 1) != (tr.dx == 0)) or ((tr.yy == 1) != (tr.dy == 0)):
+                count_by_type[paint.getFormatName() + "::weird_scale"] += 1
```

Comment:
there's nothing "weird" per se, it's just that you can't derive a PaintScaleAroundCenter with those

Reply:
But why would someone do such a thing?! It's monstrous!